src/components/formik-forms/formik-input.scss

```diff
@@ -12,17 +12,17 @@
     font-size: .875rem;
 
     &:focus {
-        box-shadow: 0 0 0 .25rem $ui-blue-25percent;
+        box-shadow: 0 0 0 .25rem $ui-purple-25percent;
         outline: none;
-        border: 1px solid $ui-blue;
+        border: 1px solid $ui-purple-dark;
         transition: all .5s ease, font-size 0s;
     }
 
     &.fail {
-        border: 1px solid $ui-orange;
+        border: 1px solid $ui-red-dark;
 
         &:focus {
-            box-shadow: 0 0 0 .25rem $ui-orange-25percent;
+            box-shadow: 0 0 0 .25rem #CC440044;
```

Comment:
would maybe be nice to stick this value in a variable or comment what its equivalent is (ui-red-25percent?) for readability